report/main.tex

```diff
@@ -26,33 +75,93 @@
 \maketitle
 
 \begin{abstract}
-\noindent Lorem ipsum dolor sit amet, consectetur adipiscing elit. Vestibulum congue gravida sem non dictum. Aenean sit amet mi fermentum ante laoreet dictum sit amet a magna. Praesent sed aliquet dui. Vivamus scelerisque condimentum mauris id euismod. Duis elementum urna eu rutrum mattis. Donec fermentum, risus et viverra aliquet, ante sapien consequat augue, sit amet dictum est nisl id elit. Proin dapibus congue tincidunt. Pellentesque habitant morbi tristique senectus et netus et malesuada fames ac turpis egestas. Fusce felis eros, aliquam sed dui ut, tempor condimentum neque. Donec quis sapien bibendum, faucibus urna sed, gravida felis. Praesent et quam ligula. Aliquam ac est eu odio tincidunt volutpat a in augue. Maecenas fermentum velit felis, vel viverra dui scelerisque pharetra. Nullam eros lorem, finibus pulvinar eleifend eu, pulvinar ac ipsum. Sed velit neque, venenatis sit amet urna vitae, vulputate ullamcorper est.
+\noindent The finite element method is one of the most widely used techniques for solving partial differential equations in various industries. There are numerous open-source software packages available that implement the finite element method. However, advancements continue to be made, particularly with the emergence of new array computing libraries such as JAX and PyTorch, which feature automatic differentiation capabilities. Additionally, there is still significant potential for innovation in the user and application interfaces of finite element software to make them more accessible for researchers. \texttt{FastFEM} is an open-source finite element software designed to provide a clean and easy-to-use Python interface while leveraging JAX for computations with its automatic differentiation features. This report presents the capabilities of \texttt{FastFEM v0.0.1}, including its mesher, solver, and plotter, along with example code snippets. The underlying theory of the finite element method is summarized, and the implementation within \texttt{FastFEM} is briefly discussed. Furthermore, the strategies employed and tools utilized to maintain high software quality are explained. Finally, conclusions are drawn, and an outlook for the future of \texttt{FastFEM} is provided.
```

Comment:
Should we note that we don't have JAX working yet?

Reply:
JAX is also discussed in the introduction. I was thinking maybe we should somehow word it so that it sounds like it's the goal but not implemented yet. Would one of you like to make these changes @sacha-escudier @int-ptr-ptr?

Reply:
I can do that after I finish my slides.

Reply:
> ...Python interface while opening the possibility to leverage JAX (in a future version) for computations with its automatic differentiation features.

could be better, but I am willing to settle for this for now.